go.mod

```diff
@@ -6,7 +6,9 @@ require (
 	github.com/algorand/go-algorand-sdk/v2 v2.0.0-20230531123858-3a0efd6a6398
 	github.com/algorand/go-codec/codec v1.1.8
 	github.com/algorand/indexer v0.0.0-20230601214318-cb2f42818f82
+	github.com/google/uuid v1.3.0
 	github.com/jackc/pgx/v4 v4.13.0
+	github.com/opensearch-project/opensearch-go v1.1.0
```

Comment:
Are you stuck with 1.1 because of our version of go?

Reply:
Good question, I initially chose it for the widest compatibility ([matrix](https://opensearch.org/docs/latest/tools/index/)), but it's now unclear why we really need it. I upgraded to use the v2 client in [f32193f](https://github.com/algorand/conduit/pull/74/commits/f32193fff02e1d0d45e4d8df5da0592879df8163).